README.md

```diff
@@ -25,6 +25,51 @@ If you want to download regularly updated builds for Android, Windows x86 and x6
 
 For game compatibility, see [community compatibility feedback](https://report.ppsspp.org/games).
 
+What's new in 1.16
+==================
+Special thanks to unknownbrackets for the new JIT compilers, and fp64 for finally cracking the vrnd instruction.
+
+- RetroAchievements support ([#17589], [#17631], many more). See [RetroAchievements on ppsspp.org](https://www.ppsspp.org/docs/reference/retro-achievements).
+- New JIT backends:
+  - RISC-V, based on IR
+  - x86 JIT based on IR. May end up faster than the existing one, already at parity.
```

Comment:
It's actually already faster in a number of cases, btw.  But there are a few things unimplemented some some games might be slower.

-[Unknown]